modules/bridge/src/main/ts/ephox/bridge/components/dialog/Input.ts

```diff
@@ -18,11 +19,13 @@ export interface Input extends FormComponentWithLabel {
   placeholder: Optional<string>;
   maximized: boolean;
   enabled: boolean;
+  id: Optional<string>;
 }
 
 const inputFields = formComponentWithLabelFields.concat([
   FieldSchema.optionString('inputMode'),
   FieldSchema.optionString('placeholder'),
+  FieldSchema.optionString('id'),
```

Comment:
What do you mean by `label pretty much ignores this value`? 
For most cases I think we will still use the label property in the input spec and not wrap it in the label. This is just for very specific cases where we want buttons or other stuff right next to an input.

Reply:
`This kind of becomes a bit low level and raw.` - I was working on adding an option for `id` attribute for more input components as well. Stuff like checkboxes, listbox, colorinput and so on. Do you think we shouldn't implement this solution across different inputs as it is an edge case and not ideal? 🤔

Reply:
If you set both `label` and `id` on a text input thing the `id` is overridden by custom label id generated by alloy. It would have been nice to now have to specify raw ids like this but rather reference things by name and generate the ids as uuids automatically but I see that it would require a lot of extra code by passing around way to query components from components by name so maybe not feasible.\
Also I wonder if there is some form of structure test we can add here that actually checks if these new options work and render the expected attributes for these components.

- [ ] Add test for structure